trace.go

```diff
@@ -144,6 +144,27 @@ func GetFields(err error) map[string]interface{} {
 	return map[string]interface{}{}
 }
 
+// UnwrapProxyField attempts to find a named field within an error returned from ReadError.
+// If found, the field is reinterpreted through json.Unmarshal call and stored in unmarshalPtr.
+// Returns true on success and false on failure.
+func UnwrapProxyField(err error, fieldName string, unmarshalPtr interface{}) bool {
```

Comment:
What do you think about calling this `UnmarshalField`?

Reply:
This is a good name, thank you!